cli/src/potpie/utility.py

```
@@ -0,0 +1,62 @@
+
+    @property
+    def celery_log_file(self):
+        return self._celery_log_file
+
+    @staticmethod
+    def base_url() -> str:
+        return os.getenv("POTPIE_BASE_URL", "http://localhost:8001")  
+
+    @staticmethod
+    def get_user_id() -> str:
+        return os.getenv("defaultUsername", "defaultuser")
```

Ruff (0.8.2)

Use capitalized environment variable `DEFAULTUSERNAME` instead of `defaultUsername`

(SIM112)